src/main/resources/avro/variantmethods.avdl

```diff
@@ -446,25 +438,9 @@ SearchCallsResponse searchCalls(
 record SearchAlleleCallsRequest {
 
   /**
-  If nonempty, only return the `AlleleCall`s in these `CallSet`s.
-  */
-  array<string> callSetIds = [];
-
-  /**
-  If nonempty, return only `AlleleCall`s in these `VariantSet`s.
-  */
-  array<string> variantSetIds = [];
-
-  /**
-  If nonempty, return only `AlleleCall`s associated with one of these `Allele`s.
-  */
-  array<string> alleleIds = [];
```

Comment:
How would you go about looking up how a particular `CallSet` calls a particular `Allele` without this field? Do you have to look at all the `AlleleCall`s?